src/main/java/org/fcrepo/migration/validator/impl/Fedora3ValidationExecutionManager.java

```diff
@@ -83,6 +88,8 @@ private void submit(final ValidationTask task) {
         executorService.submit(() -> {
             try {
                 task.run();
+            } catch (Exception ex) {
+                LOGGER.error("validation task failed {}", ex.getMessage(), ex);
```

Comment:
Should we record this failure in such a way that it shows up in the final result? Otherwise, it may get lost.

Reply:
https://jira.lyrasis.org/browse/FCREPO-3633